packages/shadycss/src/style-properties.js

```diff
@@ -495,9 +495,20 @@ class StyleProperties {
     let scope = '.' + scopeId;
     let parts = StyleUtil.splitSelectorList(selector);
     for (let i=0, l=parts.length, p; (i<l) && (p=parts[i]); i++) {
-      parts[i] = p.match(hostRx) ?
-        p.replace(hostSelector, scope) :
-        scope + ' ' + p;
+      if (p.match(hostRx)) {
+        parts[i] = p.replace(hostSelector, scope);
```

Comment:
Suggest changing the name `parts` to something else since it's too easily confused with shadow part.

Reply:
Done (went with "segments").